### PR TITLE
CONCOCT: Update versions, replace scripts and tests

### PR DESCRIPTION
**Pull request by bebatut**
FOR CONTRIBUTOR:
* [X] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [X] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

File: tools/concoct/merge_cut_up_clustering.xml [hunk 1]
```diff
@@ -1,32 +1,36 @@
...
-python '$__tool_directory__/merge_cut_up_clustering.py'
---input '$input'
---output '$output'
+merge_cutup_clustering.py
+    '$cutup_clustering_result'
+    > '$output'
     ]]></command>
     <inputs>
-        <param name="input" type="data" format="tabular" label="Tabular file containng cut up contigs"/>
+        <param name="cutup_clustering_result" type="data" format="csv" label="Clusters generated by CONCOCT"/>
```
Comment (bgruening) [inline tools/concoct/merge_cut_up_clustering.xml:13]: should we convert that to tabular?

**Comment by gregvonkuster** [comment]
Other than my question about eliminating the python call in the command line, these changes look good to me.  Thanks @bebatut !

File: tools/concoct/concoct_coverage_table.xml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <requirement type="package" version="1.15.1">samtools</requirement>
+    </expand>
+    <command detect_errors="exit_code"><![CDATA[
+mkdir 'mapping' &&
+#for $e in $bamfiles
+    ln -s '$e' 'mapping/${e.element_identifier}.sorted.bam' &&
+    samtools index 'mapping/${e.element_identifier}.sorted.bam' 'mapping/${e.element_identifier}.bam.bai' &&
+    mv 'mapping/${e.element_identifier}.sorted.bam' 'mapping/${e.element_identifier}.bam' &&
+#end for
+concoct_coverage_table.py
```
[the remark this answers is not in the document]
Comment (gregvonkuster) [reply tools/concoct/concoct_coverage_table.xml:16]: Ok, good to know - thanks!

[the remark this answers is not in the document]
Comment (gregvonkuster) [reply tools/concoct/concoct_coverage_table.xml:16]: Ah, very cool!

[the remark this answers is not in the document]
Comment (bgruening) [reply tools/concoct/concoct_coverage_table.xml:16]: You would need to update the bioconda package and the Galaxy wrapper.

**Review by bgruening — APPROVED** [review]

**Pull request merged**